README.md

```diff
@@ -88,6 +89,22 @@ lru_redux thread safe   2.190000   0.010000   2.200000 (  2.185512)
 
 ```
 
+#### TTL
+```
+$ ruby ./bench/bench_ttl.rb
+Rehearsal -----------------------------------------------------------------------
+FastCache                             6.240000   0.070000   6.310000 (  6.302569)
+LruRedux::TTL::Cache                  4.700000   0.010000   4.710000 (  4.712858)
```

Comment:
we should make sure we properly link to all these other cache alternatives from the readme. FastCache have some interesting documentation for example.